### ISSUE TITLE
Add out-of-the-box/batteries-included demo wrappers

### ISSUE THREAD

**Issue by davelab6** [w3c/PFE-analysis#96]
(Based on a chat with @garretrieger)

This repo contains a c++ client and server implementation of patch subset; it's being wrapped by the GF team's internal web framework to drive this demo:

https://fonts.gstatic.com/experimental/patch_subset/demo.html

While this repo is missing a http server that calls into the code, but otherwise this can basically be used as-is. 

Likewise for the client, it is missing a little bit of javascript to wrap the wasm library (found in the js_client folder)

So, if you want to replicate the GF demo, you "just" need to write a wrapper in your http server of choice to call the server handle method (seen here: https://github.com/w3c/PFE-analysis/blob/main/patch_subset/patch_subset_server.h#L18) and return the result

So that this repo is not "missing" anything, I'd like to propose providing a simple "out of the box" demo with the 2 wrappers, eg a `python -m SimpleHTTPServer` script or a `node` script

**Comment by yisibl** [comment]
I am very interested in this, do you have any progress?

It seems to have moved here: https://github.com/w3c/patch-subset-incxfer/blob/main/patch_subset/patch_subset_server.h

**Comment by garretrieger** [comment]
That's right, the server/client implementation are currently being updated to move from the prototype implementation to one that matches the draft spec. No immediate plans to add the wrapper + demo until we have that completed.
